### ISSUE TITLE
It's jarring how class info moves from right sidebar to left sidebar

### ISSUE THREAD

**Issue by tvolkert** [dart-lang/dartdoc#2616]
Looking at https://api.flutter.dev/flutter/dart-ui/Rect-class.html, you see the following, with class constructors, properties, methods, operators, etc. on the right sidebar:

<img width="981" alt="Screen Shot 2021-04-18 at 11 22 10 AM" src="https://user-images.githubusercontent.com/15253456/115156314-81217b80-a038-11eb-9765-85c549ea2869.png">

But then if you click to see details for any of those properties, that same info moves to the left sidebar -- and the class listing that was on the left sidebar goes away:

<img width="986" alt="Screen Shot 2021-04-18 at 11 22 23 AM" src="https://user-images.githubusercontent.com/15253456/115156342-9c8c8680-a038-11eb-9e54-7a6688e39aa0.png">

This is a very jarring user experience.

**Comment by jcollins-g** [comment]
Right now in general sidebars move from right to left as one drills down.  To correct this, we'll have to redesign how sidebars work (perhaps, by not having two of them and using expanding/collapsing sections or other techniques).

**Comment by guidezpl** [comment]
Yeah, I also found this confusing. A bad solution to this would be to translate pages on the X axis when navigating.

Given you have a table of contents for 
- the current page on the right
- the enclosing thing (class, library, etc.) on the left

I think a better solution would be to remove the latter, and only show the table of contents for the current page. (either on the left or right). What use is it to me to know, when browsing the `Rect` docs, that the same library provides a `ColorFilter` class? Not that much. If I want to know what the enclosing thing contains, I can simply use the breadcrumbs to navigate up one level and see a [more complete page](https://api.flutter.dev/flutter/dart-ui/dart-ui-library.html) than what is currently shown on the left.

Would you have a preference for a single left or right TOC?
